examples/react-ssr-example/package.json

```diff
@@ -84,31 +82,30 @@
     "node": "8.x || 9.x || 10.x"
   },
   "scripts": {
-    "cli": "pixeloven-cli",
+    "cli": "pixeloven",
     "build": "run-p build:app build:story",
-    "build:app": "pixeloven-webpack build",
-    "build:story": "pixeloven-storybook build",
-    "prebuild": "pixeloven-tasks compile:clean",
-    "clean": "pixeloven-tasks clean",
-    "development": "pixeloven-webpack-dev-server start",
-    "document": "pixeloven-tasks document:ts src",
-    "predocument": "pixeloven-tasks document:clean",
+    "build:app": "pixeloven webpack build",
+    "build:story": "pixeloven storybook build",
+    "prebuild": "pixeloven delete dist",
+    "development": "pixeloven webpack start",
+    "document": "pixeloven document ts ./src",
+    "predocument": "pixeloven delete docs",
     "env:docker": "cp .env.docker .env",
     "env:local": "cp .env.localhost .env",
     "env:production": "cp .env.localhost .env",
     "generate": "pixeloven-generators generate",
     "lint": "run-p lint:ts lint:scss",
-    "lint:ts": "pixeloven-tasks lint:ts src/**/*.{ts,tsx}",
-    "lint:scss": "pixeloven-tasks lint:scss src/**/*.{scss}",
-    "pretest": "pixeloven-tasks test:clean",
-    "pretty": "pixeloven-tasks pretty src/**/*.{scss,ts,tsx}",
-    "pretty:ts": "pixeloven-tasks pretty:ts src/**/*.{ts,tsx}",
-    "pretty:scss": "pixeloven-tasks pretty:scss src/**/*.{scss}",
+    "lint:ts": "pixeloven lint ts ./src/**/*.{ts,tsx}",
```

Comment:
Should we think about an `@root` shorthand for stuff like this?

Reply:
I don't believe those will work in this CLI context. We luckily support globbing in this pattern which automatically gets parsed into individual file paths matching that pattern